src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -171,6 +172,58 @@ public bool CreateServiceMetadata(ServiceMetadata serviceMetadata)
             return true;
         }
 
+        /// <summary>
+        /// Creates the application meta data file
+        /// </summary>
+        /// <param name="applicationOwnerId">the application owner</param>
+        /// <param name="applicationId">the application id</param>
+        public void CreateApplicationMetaData(string applicationOwnerId, string applicationId)
```

Comment:
Metadata to be consistent with the rest of the solution

Reply:
Fixed